vue-components/tests/e2e/specs/Dialog.test.js

```diff
@@ -0,0 +1,117 @@
+				.pause( 500 )
+				.waitForElementPresent( '.wikit-Dialog' )
+				.sendKeys( 'body', client.Keys.TAB )
+				.elementActive( function ( result ) {
+					client.elementIdText( getWebElementIdFromActiveElementResult( result ), ( element ) => {
+						const elementText1 = element.value.replace( /\s+/g, ' ' ).trim();
+						client.assert.equal( elementText1, 'Primary action' );
+					} );
+				} )
+				// sending multiple keys inside array will fail in Firefox.
+				// See: https://github.com/nightwatchjs/nightwatch/issues/2468#issuecomment-738089219
+				.sendKeys( 'body',
+					currentBrowser !== 'firefox' ?
+						client.Keys.TAB + client.Keys.TAB : client.Keys.TAB )
+				.elementActive( function ( result ) {
+					client.elementIdText( getWebElementIdFromActiveElementResult( result ), ( element ) => {
+						const elementText2 = element.value.replace( /\s+/g, ' ' ).trim();
+						client.assert.equal( elementText2, 'Secondary action' );
+					} );
+				} )
+				// different browsers are counting the combinations differently.
+				// firefox starts in the previous selected element,
+				// while chrome starts the tab count on each elementActive call
+				// chrome: makes sure it goes back to the first element after four tabs
+				.sendKeys( 'body', currentBrowser !== 'firefox' ?
+					client.Keys.TAB + client.Keys.TAB + client.Keys.TAB + client.Keys.TAB
+					: client.Keys.TAB + client.Keys.TAB )
+				.elementActive( function ( result ) {
+					client.elementIdText( getWebElementIdFromActiveElementResult( result ), ( element ) => {
+						const elementText3 = element.value.replace( /\s+/g, ' ' ).trim();
+						client.assert.equal( elementText3, 'Primary action' );
+					} );
+				} );
+		}
+	} );
+
+	it( 'prevents underlying page from scrolling when opened and on initial render', function ( client ) {
+		if ( client.options.desiredCapabilities.browserName !== 'internet explorer' ) {
+			client
+				.execute( 'document.body.style.height = "150vh";' )
+				.execute( 'window.scrollTo( 0, document.body.scrollHeight );' )
+				.execute( 'return document.documentElement.scrollHeight > document.documentElement.clientHeight;',
+					function ( result ) {
+						this.assert.ok( result.value === true, 'body shows scrollbars' );
+					} )
+				.click( '.wikit-Button' )
+				.pause( 250 )
+				.execute( 'return document.documentElement.scrollHeight > document.documentElement.clientHeight;',
+					function ( result ) {
+						this.assert.not.ok( result.value === false, 'body does not show scrollbars' );
+					} );
+		}
+	} );
+
+	it( 'resets dialog scroll bars to top when closed and reopened', function ( client ) {
+		if ( client.options.desiredCapabilities.browserName !== 'internet explorer' ) {
+			client
+				.click( '.wikit-Button' )
+				.pause( 500 )
+				.waitForElementPresent( '.wikit-Dialog' )
+				// substract 100px from the dialog's original height so scrollbars are shown
+				// TODO: see why this is failing in IE11 in saucelabs
```

Comment:
We know that it fails because the whole Dialog doesn't load, right?

Reply:
yep. forgot to check the comments. :D